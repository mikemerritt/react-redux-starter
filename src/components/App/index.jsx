import React, { PropTypes } from 'react';
import Helmet from 'react-helmet';
import DevTools from '../../utils/devtools';

class App extends React.Component {
  static propTypes = {
    children: PropTypes.oneOfType([
      React.PropTypes.arrayOf(React.PropTypes.node),
      React.PropTypes.node
    ])
  };

  render() {
    return (
      <div>
        <Helmet title="react-redux-starter" />
        {this.props.children}
        <DevTools />
      </div>
    );
  }
}

export default App;
