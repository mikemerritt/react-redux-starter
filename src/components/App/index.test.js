import React from 'react';
import renderer from 'react-test-renderer';
import App from './index';

// Example test.
describe('App', () => {
  it('renders (example test)', () => {
    const component = renderer.create(
      <App />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
