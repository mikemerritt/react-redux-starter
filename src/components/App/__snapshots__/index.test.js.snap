// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders (example test) 1`] = `<div />`;
